pkg/tsdb/azuremonitor/azure-resource-graph-datasource.go

```diff
@@ -178,7 +179,9 @@ func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, query *
 	if err != nil {
 		return dataResponseErrorWithExecuted(err)
 	}
-	url := azurePortalUrl + "/#blade/HubsExtension/ArgQueryBlade/query/" + query.InterpolatedQuery
+	frameLink := strings.ReplaceAll(query.InterpolatedQuery, "/", "%2F")
+	frameLink = strings.ReplaceAll(frameLink, "\n", "%0A")
+	url := azurePortalUrl + "/#blade/HubsExtension/ArgQueryBlade/query/" + frameLink
```

Comment:
Could you use `url.QueryEscape()` instead of manually escaping?

Reply:
`url.QueryEscape(query.InterpolatedQuery)` has some issues. It populated all the spaces with "+".

Reply:
Ahhh, perhaps [`PathEscape`](https://pkg.go.dev/net/url#PathEscape) is the better choice here? I'm not too familiar with Go, so I might have to ask @andresmgot or @sunker to take a look with a review here.

The concern that I have is that the value is not properly escaped and other queries will silently fail to link properly without us knowing.

Reply:
PathEscape works!